Intro.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+
```

Comment:
What is the use of this Intro component? Are we using it only for snapshot tests?
If yes, should we have it?